wiki/Twitter/pl.md

```diff
@@ -1,17 +1,24 @@
----
-outdated_translation: true
----
-
 # Twitter
 
-## My także tweetujemy!
+osu! posiada kilka kont na Twitterze; każde z nich pełni inne funkcje. Pomimo niewielkiej aktywności oferują one wystarczające informacje o tym, co dzieje się w świecie osu!.
+
+## Serwisy
```

Comment:
i dont think it really matches with the original text (`## Service`), maybe it should be something like this instead?

```suggestion
## Usługi
```

Reply:
yeah, thanks :)